AirCasting/Graph/AirCastingGraphSimplifier.swift

```diff
@@ -2,16 +2,15 @@
 //
 
 import Foundation
-import SwiftSimplify
 import Charts
 
 final class AirCastingGraphSimplifier {
     
-    static func simplify<P: Point2DRepresentable>(points: [P], visibleElementsNumber: Int, thresholdLimit: Int) -> [P] {
+    static func simplify(points: [ChartDataEntry], visibleElementsNumber: Int, thresholdLimit: Int) -> [ChartDataEntry] {
```

Comment:
is this de-generalization really needed? It doesn't look like it is changing anything except, well, de-generalizing ;)

Reply:
Yeah, you right. There is one thing - the  `Point2DRepresentable` was taken from SwiftSimplifier library which we have added. I thought that if we are going to use our own implementation now, we can get rid of the library in the future. That was the point of changing it, what do you think?

Reply:
We could create our own protocol for that, but in this case I think it wouldn't matter. Cool then! 😀